### ISSUE TITLE
Windows: Bumping to Bazel 3.2.0+ will break Windows toolchain config generation

### ISSUE THREAD

**Issue by sunjayBhatia** [envoyproxy/envoy-build-tools#77]
See: https://github.com/bazelbuild/bazel-toolchains/pull/886

We can start using Bazel 3.2.0+ once this PR is merged into bazel-toolchains

**Comment by sunjayBhatia** [comment]
cc @wrowe @davinci26 @nigriMSFT
